packages/bpk-svgs/gulpfile.babel.js

```diff
@@ -339,7 +358,7 @@ gulp.task('icons-font', ['icons-common'], () => {
     mappingStream.end();
   });
 
-  return merge(saveFont, saveMapping);
+  return merge.apply(null, [...saveFonts, saveMapping]);
```

Comment:
Why not just `merge(...saveFonts, saveMapping)`?

Reply:
I had doubts about that method, but never tried it